pkg/cfn/manager/nodegroup.go

```diff
@@ -137,15 +152,70 @@ func (c *StackCollection) ScaleNodeGroup(id int) error {
 	return c.UpdateStack(name, "scale-nodegroup", descriptionBuffer.String(), []byte(template), nil)
 }
 
-func (c *StackCollection) getStackTemplate(stackName string) (string, error) {
-	input := &cfn.GetTemplateInput{
-		StackName: aws.String(stackName),
+// GetNodeGroupSummaries returns a list of summaries for the nodegroups of a cluster
+func (c *StackCollection) GetNodeGroupSummaries() ([]*NodeGroupSummary, error) {
+	stacks, err := c.ListStacks(fmt.Sprintf("^(eksctl|EKS)-%s-nodegroup-\\d+$", c.spec.ClusterName), cfn.StackStatusCreateComplete)
+	if err != nil {
+		return nil, errors.Wrap(err, "getting nodegroup stacks")
+	}
+
+	summaries := []*NodeGroupSummary{}
+	for _, stack := range stacks {
+		logger.Info("stack %s\n", *stack.StackName)
+		logger.Debug("stack = %#v", stack)
+
+		template, err := c.GetStackTemplate(*stack.StackName)
+		if err != nil {
+			return nil, errors.Wrapf(err, "error getting Cloudformation template for stack %s", *stack.StackName)
+		}
+
+		//TODO: create a map function
+		seq := getNodeGroupID(stack.Tags)
```

Comment:
@richardcase Would you mind clarifying a bit more here?

I have `get nodegroup` working like this in my branch:

```
$ ./eksctl get nodegroup --cluster adorable-sheepdog-1543570021
2018-12-04T16:46:41+09:00 [ℹ]  stack eksctl-adorable-sheepdog-1543570021-nodegroup-0
STACKNAME					SEQ	CREATED			MIN SIZE	MAX SIZE	DESIRED CAPACITY	INSTANCE TYPE	IMAGE ID
eksctl-adorable-sheepdog-1543570021-nodegroup-0	0	2018-12-04T07:21:34Z	2		2		2			m5.large	ami-0f54a2f7d2e9c88b3
```

But I required no change around this TODO.